.github/workflows/ci.yml

```diff
@@ -1076,6 +1076,7 @@ jobs:
 
       - name: Build Antora Docs
         run: |
+          set -xe
```

Comment:
TIL you need to set this for multi-line `run` commands.

This is good to know; thank you.